content/en/apps/examples/supervisor-reference-app.md

```diff
@@ -0,0 +1,72 @@
+
+## Solution overview
+The supervisor reference app workflows enables supervisors to: provide quality assurance supervisory activities, plan and document CHW supervisory activities, support CHW activities and monitor CHW performance in real time. The supervisor reference app has been designed to:
+<ul>
+<li>Monitor the performance of the CHW and supervisors using the in-app aggregate target and the supervisor targets functionalities respectively.</li>
+
+<li>Document and track CHW and supervisory activities.</li>
+<li>Schedule CHW monthly meetings quality monitoring visits and follow up visits.</li>
+<li>Support supervisors to report on health outcomes for their supervisory area.</li>
+<li>Identify CHWs who have not been visited by a supervisor.</li>
+<li>Follow up on CHW who have been inactive for 3 months.</li>
+</ul>
+
+## Forms hierarchy
+The diagram indicates forms and tasks that can be filled by a supervisor on the supervisor reference app. Some of the forms are accessible as actions on the supervisor and CHW profiles while others are accessible as tasks for a supervisor and a CHW.
+
+{{< figure src="Forms_hierarchy.png"  link="Forms_hierarchy.png" alt="Forms hierarchy" title="" class="left col-12">}}
+
+## Supervisor app workflows and features.
+
+### CHW monthly meeting workflows.
+A Supervisor conducts a CHW monthly meeting with all the CHWs. During the meeting the supervisor mentors the CHWs, reviews the CHWs performance and discusses with the CHWs the challenges they might be facing in their provision of community health services.
+
+{{< figure src="Monthly_meeting_workflows.png"  link="Monthly_meeting_workflows.png" alt="Monthly meeting workflows" title="" class="left col-10">}}
+
+### Quality monitoring workflow
+With the supervisor reference app, a supervisor can schedule a quality monitoring visit to assess the quality of services being provided by a CHW. During the quality monitoring visit a supervisor shadows CHWs as they provide services to household members. A supervisor can schedule additional quality monitoring follow up visits for CHWs who are identified to have some weak areas and may need further support and mentorship from the supervisor.
+
+{{< figure src="Qualtiy_monitoring_workflows.png"  link="Qualtiy_monitoring_workflows.png" alt="Qualtiy monitoring workflows" title="" class="left col-10">}}
```

Comment:
please convert this image to a markdown table.

be sure new images don't have "rectangular snip" ghost image:

![image](https://user-images.githubusercontent.com/8253488/194175041-bddfef6d-5f6c-4d31-8529-2ee9122ddab0.png)

Reply:
Hi @mrjones-plip, I am following [documentation on style guide](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#images) and little confused on using markdown table. Could you please advise?

Reply:
Sure! 

It won't be as quite as good looking as the image, but just a 6 column table would likely do.  Also, I see you're missing 3c?  Maybe rename 3d -> 3c?  

| | ![flow1](https://user-images.githubusercontent.com/8253488/197227565-e267dffc-70e7-46f6-a1e0-1130438f5de3.png) | --> |  ![flow2](https://user-images.githubusercontent.com/8253488/197227564-aa51655c-ee42-494b-83bb-7df56e3f7fee.png)  |  --> | ![flow3](https://user-images.githubusercontent.com/8253488/197227559-b77b00b1-0059-48d3-9030-3a4bbb2f0341.png)|
|--|--|--|--|--|--|
| **1A** | Supervisor Checks/Monitors... | **2A** | Quality monitor planning... | **3A**  |  Quality monitoring task... <br /><br /> Shadowing reminder...  |
| **1B** | Supervisor UHC mode... | | | **3B** | Quality MOnitoring form... |

Reply:
thanks @mrjones-plip 
cc @antonykhaemba